### DISCUSSION TITLE
HandleBars.Compile replaces '&' with '&amp;' in .NET. How can i prevent such conversions in compilation in .NET?

### DISCUSSION THREAD

**Discussion by SoniJai** [Handlebars-Net/Handlebars.Net#463] [Q&A]
`string location= "Location {{LocationName}}";`

`var data = new {LocationName = "Ohio & Utah"};`

```
var templateFunc = Handlebars.Compile(location);
var message = templateFunc(data);
return message;
```

```
output is : Location Ohio &amp; Utah
```

**Comment by haas-daniel** [comment]
You can use the "triple-stash" {{{ }}} in your template if you don't want handlebars to escape the value:
`string location= "Location {{{LocationName}}}";`

https://handlebarsjs.com/guide/expressions.html#html-escaping
